code/src/components/Album.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+export const Album = (props) => {
+  return (
```

Comment:
In general it's well done component, but don't you think it could be split into smaller components?
I would say that if you see your JSX takes more than 15 lines of code, then it's a clear info that something could be divided.
It's not an bug on your side, it's more about how we divide our code into logical, separate parts, just making this clear :)